### DISCUSSION TITLE
`brew update` output suggestion

### DISCUSSION THREAD

**Discussion by ywwry66** [Homebrew/discussions#3350] [Everyday usage]
Hi there. Just a quick suggestion for the new default `brew update` output behavior. Since `brew update` now only prints all outdated formulae that user has installed, this informative sentence
```
You have 1 outdated formula installed.
You can upgrade it with brew upgrade
or list it with brew outdated.
```
can also be shortened to
```
You have 1 outdated formula installed.
You can upgrade it with brew upgrade.
```
I don't see any benefit to run `brew outdated` right after `brew update` now.

**Comment by gwww** [comment]
I partially agree. Right now the output of doing a `brew update` is as follows (for example):
```
Updated Homebrew from a3d14ffb9 to b40b73d0d.
Updated 2 taps (homebrew/core and homebrew/cask).
==> Outdated Formulae
asdf                                                       gh                                                         lua-language-server

You have 3 outdated formulae installed.
You can upgrade them with brew upgrade
or list them with brew outdated.
```
What would be even better is if the output was like the following:
```
Updated Homebrew from a3d14ffb9 to b40b73d0d.
Updated 2 taps (homebrew/core and homebrew/cask).
==> Outdated Formulae
asdf (0.10.1) < 0.10.2
gh (2.11.3) < 2.12.1
lua-language-server (3.2.4) < 3.2.5

You have 3 outdated formulae installed. You can upgrade them with brew upgrade.
```
Edit: even with the new output format I still do a `brew outdated` right after a `brew update` so that I can see the change in versions.

**Reply by ywwry66** [reply]
You are right. Another difference between the outputs of `brew update` and `brew outdated` is, `brew update` only lists the outdated formulae introduced by the most recent pull from the tap repos while `brew outdated` lists all. So it still makes sense to say
```
Updated Homebrew from a3d14ffb9 to b40b73d0d.
Updated 2 taps (homebrew/core and homebrew/cask).
==> Outdated Formulae
asdf (0.10.1) < 0.10.2

You have 3 outdated formulae installed.
You can upgrade them with brew upgrade
or list them with brew outdated.
```
as in your example.
